### PR TITLE
Feature 1 to Dev with CI yml

### PR DESCRIPTION
**Pull request by zawnyein**

### CODE REVIEW

File: .github/workflows/feature.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: CICD for Group 2 Chat Application - Development
```
Comment (Mha47) [inline .github/workflows/feature.yml:1]: ok

**Comment by Mha47** [comment]
is ok

**Comment by zawnyein** [comment]
its ok

File: .github/workflows/feature.yml
Comment (Mha47) [inline .github/workflows/feature.yml:1]: ok

File: .github/workflows/uat.yml
Comment (Mha47) [inline .github/workflows/uat.yml:1]: ok

File: environments/uat/backend.tf
Comment (Mha47) [inline environments/uat/backend.tf:1]: ok

File: environments/uat/variables.tf
Comment (Mha47) [inline environments/uat/variables.tf:1]: ok

**Review by Mha47 — APPROVED** [review]
All ok bro

**Pull request merged**